pybamm/parameters/lithium_ion_parameters.py

```diff
@@ -628,22 +628,26 @@ def _set_dimensionless_parameters(self):
         self.T_init = self.therm.T_init
         self.c_e_init = self.c_e_init_dimensional / self.c_e_typ
 
-    def chi(self, c_e):
+    def chi(self, c_e, T=0):
```

Comment:
Only when `dlnf_dlnc` is enabled, `chi` becomes a function of temperature. Otherwise and in the existing examples, `chi` is independent on temperature.

Reply:
That is fine, in the existing examples where `chi` is independent of temperature it should still be passed `T`, but then ignore it and return a function that doesn't depend on `T`. Like [this example](https://github.com/pybamm-team/PyBaMM/blob/develop/pybamm/input/parameters/lithium-ion/electrolytes/lipf6_Nyman2008/electrolyte_conductivity_Nyman2008.py)

Reply:
This is a big movement, because `chi()` has been used in many other places. I will leave it like this and we can modify it if there are more parameter sets considering it temperature dependent.

Reply:
I think it would be good to have `chi(c,T)` everywhere for consistency. at the moment even though `chi` can depend on `T` it doesn't get called properly everywhere

Reply:
OK, I have made all modifications.